ebl/v3/issuance/EBL_ISS_v3.0.0-Beta-2.yaml

```diff
@@ -757,6 +757,8 @@ components:
             - `SECR` (Secro)
             - `TRGO` (TradeGO)
             - `ETEU` (eTEU)
+            
+            Must be a code this list [GitHub](https://github.com/dcsaorg/DCSA-OpenAPI/blob/master/reference-data/eblsolutionproviders-v1.csv).
```

Comment:
Can we please use `v3.0.0-Beta-2` in that filename, so it is trivial to know which version of the standard it applies to?

(We will need it in the CTK as well and there we would end up having to use `v3.0.0-Beta-2` for technical reasons)

Reply:
Fixed